spec.md

```diff
@@ -193,6 +214,53 @@ To “close” a node $N_i$:
 
 # Rolling Hash Functions
 
+## CP32
+
+The `cp32` hash function is based on cyclic polynomials. The family of
+related functions is sometimes also called "buzhash." `cp32` is the
+recommended hash function for use with hashsplit; use it unless you have
+clear reasons for doing otherwise.
+
+### Definition
+
+We define the function $\operatorname{CP32} \in V_8 \rightarrow U_{32}$
+as:
+
+$\operatorname{CP32}(X) = \bigoplus_{i = 0}^{|X| - 1}
+\operatorname{ROT}_L(g(X_i), |X| - i + 1)$
+
+Where $g(n) = G_n$ and the sequence $G \in V_{32}$ is defined in the
```

Comment:
Why define g(n) at all? Why not simply use G_n wherever you're using g(n)?

(Is it because you're trying to avoid a subscript of a subscript? If so, that has implications for [my PR](https://github.com/hashsplit/hashsplit-spec/pull/23).)

Reply:
Yeah, it was just to avoid the double subscript, though I don't feel incredibly strongly.